src/components/Button/Button.tsx

```diff
@@ -100,8 +100,9 @@ export const ButtonGrey = styled(Base)`
   border: 1px solid #2a2f42;
   background-color: ${props => props.theme.bg1And2};
   color: ${({ theme }) => theme.text5};
-  font-size: 16px;
-  padding: ${({ padding }) => (padding ? padding : '10px')};
+  font-size: 10px;
```

Comment:
isn't it impact other components which are using ButtonGray?

Reply:
Yes. It should be removed.

Reply:
Fixed. We should also refactor these buttons, look to their usecase and uniform.